test/e2e/Controller/Api/UserController.test.ts

```diff
@@ -119,4 +119,30 @@ describe("UserModel controller - edit account", async () => {
             done();
         }
     });
+
+    it("Should return user account data", (done) => {
```

Comment:
Write test where you pass non existing user and that should send response 404

Reply:
User ID is fetched from middleware, which is the currently logged in user so a 404 can't happen here

Reply:
Oh, sorry, thought you are not catching the SeviceError, nevermind